Finished_Pluggins/Schedule_Tables.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/* 
+Plugin Name: Schedule Table
```

Comment:
Add a line for the description and one for the author-
Description: <description of plugin>
Author: Jacob Tower